install_requirements.py

```diff
@@ -78,6 +78,14 @@ def python_is_compatible():
         else:
             print(f"Error: {arg} must follow --pybind")
             sys.exit(1)
+    elif arg == "--clean":
+        print(f"Cleaning build artifacts...")
+        if sys.platform == "win32":
+            subprocess.run(["rmdir", "/s", "cmake-out*/"], check=True)
```

Comment:
Sounds like rmdir can't handle wildcards: https://superuser.com/a/764355

https://docs.python.org/3/library/shutil.html#shutil.rmtree is portable way to do this